x-pack/plugins/reporting/server/lib/store/mapping.ts

```diff
@@ -41,9 +32,10 @@ export const mapping = {
     },
   },
   browser_type: { type: 'keyword' },
+  migrationVersion: { type: 'keyword' }, // new field to distinguish reports that were scheduled with Task Manager
```

Comment:
I could not find `migrationVersion` being used in `reporting` plugin.

Reply:
But there is `migration_version` used in few places.

Reply:
Thank you for noticing this - the actual field used in the code is `migration_version`. This part needed to be changed to map an actual field :)

Reply:
d9cd73f05ed7